arlo-client/src/components/Audit/index.tsx

```diff
@@ -79,58 +81,105 @@ const Audit: React.FC<{}> = () => {
   const [menuItems, refresh] = useSetupMenuItems(stage, setStage, electionId)
 
   useEffect(() => {
-    refresh()
-  }, [refresh])
+    if (
+      isAuthenticated &&
+      viewMatch === 'setup' &&
+      meta!.type === 'audit_admin'
```

Comment:
might we have an easier time if we fully separated the UX between audit admin and jurisdiction admin? I wonder if these internal if statements will be harder to manage over time.

Reply:
@benadida Yes, that's actually being fixed here: https://github.com/votingworks/arlo/pull/464